### DISCUSSION TITLE
⭐ FEAT: Enhance UI of the DOCS section

### DISCUSSION THREAD

**Discussion by Shubh8899** [ArslanYM/StarterHive#240] [Ideas]
Hey @ArslanYM for #218 i would like UI of docs to be something like this (see attached screenshot). 

Obviously following the theme of application in dark mode & light mode. Also instead of Screenshot images like it is in current Docs we could use generated images as shown in attached screenshot.  

<img width="1022" alt="Screenshot 2023-06-10 at 7 36 56 PM" src="https://github.com/ArslanYM/StarterHive/assets/63413220/fa9e1466-695a-460d-9681-737c94021fdf">

**Comment by ArslanYM** [comment]
this looks sick!!! I would love this!!

**Reply by Shubh8899** [reply]
If you have any suggestion we can discuss or I can start working on it. 
I will create images to use in Docs page first & then work on UI.

**Reply by ArslanYM** [reply]
Keep the UI as minimalist as possible and dynamic for both light/dark mode.
